### PR TITLE
Add dantro

### PR DESCRIPTION
**Pull request by blsqr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
   - [x] @jeremiastraub
   - [x] @herdeanu

### CODE REVIEW

**Comment by blsqr** [comment]
@herdeanu @jeremiastraub Could you confirm that you accept to be a maintainer for this recipe? (See checklist.)

**Comment by jeremiastraub** [comment]
I accept to be maintainer for this recipe.

**Comment by herdeanu** [comment]
I also accept to be maintainer.

**Comment by blsqr** [comment]
@conda-forge/help-python This would be ready for review. Thanks!

File: recipes/dantro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dask >=2.10.1
+    - distributed >=2.10.0
+    - h5py >=2.10.0
+    - matplotlib >=3.1.3
```
Comment (scopatz) [inline recipes/dantro/meta.yaml:26]: This should be `matplotlib-base`

Comment (blsqr) [reply recipes/dantro/meta.yaml:26]: Thanks!
```suggestion
    - matplotlib-base >=3.1.3
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**